mgradm/cmd/install/podman/ssl.go

```diff
@@ -95,6 +96,27 @@ func generateSSLCertificates(image string, flags *adm_utils.ServerFlags, fqdn st
 }
 
 const sslSetupScript = `
+	getMachineName() {
+	  hostname="$1"
+
+	  hostname=$(echo "$hostname" | sed 's/\*/_star_/g')
```

Comment:
```suggestion
	  hostname="${hostname//\*/_star_)"
```

Should work too (assuming we call this script using bash)

```bash
$ X="*.test.*.com"
$ echo ${X//\*/_star_}
_star_.test._star_.com
```

Reply:
We are calling it with `sh`